src/app/views/functions/show.html.erb

```diff
@@ -1 +1,2 @@
+<%= content_for :title, sprintf("%s - by: %s", @function.name, @user.name) %>
```

Comment:
you can set the content to an array here, then in the helper prepend to the provided array and join them with a separator like ">>" the small arrows, or `-` or any other separator, that will remove the `if...else` and make this line as follows
```suggestion
<%= content_for :title, [@function.name, @user.name] %>
```